microprofile/microprofile-fault-tolerance-2.0/src/test/java/org/wildfly/swarm/ts/microprofile/fault/tolerance/v20/FaultTolerance20AsyncTest.java

```diff
@@ -0,0 +1,175 @@
+package org.wildfly.swarm.ts.microprofile.fault.tolerance.v20;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Queue;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.http.client.fluent.Request;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.wildfly.swarm.arquillian.DefaultDeployment;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+@RunWith(Arquillian.class)
+@DefaultDeployment
+public class FaultTolerance20AsyncTest {
+    // Add support of the CompletionStage return type when annotated with @Asynchronous (#110).
```

Comment:
What does this comment mean? It is about `CompletionStage` being a new feature of MP FT 2.0?

Reply:
forgotten comments